cli/kubectl_kadalu/utils.py

```diff
@@ -3,23 +3,28 @@
 """
 
 import subprocess
-
+#import sys
 KUBECTL_CMD = "kubectl"
 
+#class CmdResponse: # noqa # pylint: disable=too-few-public-methods
 
-class CmdResponse: # noqa # pylint: disable=too-few-public-methods
+class CmdResponse(object): # noqa # pylint: disable=too-few-public-methods
     """ Class for checking the response """
-    def __init__(self, rc, out, err):
-        self.returncode = rc
+    def __init__(self, returncode, out, err):
+        self.returncode = returncode
         self.stdout = out
         self.stderr = err
 
 
 class CommandError(Exception):
     """ Class for handling exceptions """
-    def __init__(self, rc, err):
-        super().__init__("error %d %s" % (rc, err))
-        self.returncode = rc
+    def __init__(self, returncode, err):
+        # if sys.version_info >= (3,):
+        #     super().__init__("error %d %s" % (returncode, err))
+        # else:
+        #     super(CommandError, self).__init__(u"error %d %s" % (returncode, err))
+        super(CommandError, self).__init__(u"error %d %s" % (returncode, err))
```

Comment:
is this working for Python3 as well?

Reply:
yes anna its working
```
vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ pylint3 utils.py
No config file found, using default configuration

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ pylint utils.py 
No config file found, using default configuration

------------------------------------
Your code has been rated at 10.00/10

vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ python utils.py
vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ python3 utils.py
```

Reply:
Just try to run using Python3 once. (May be by changing the kubectl command to something else in code, which will cause CommandError)

Reply:
utils.py compiled on both ver of py . main.py has import errors without the changes